kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/clients/KafClient.java

```diff
@@ -53,24 +51,15 @@ public void produceMessages(String topicName, String bootstrap, String message,
                 "--image=" + Constants.KAF_CLIENT_IMAGE,
                 "--", "kaf", "-n", String.valueOf(numOfMessages), "-b", bootstrap, "produce", topicName);
 
-        LOGGER.atInfo().setMessage("Executing command: {} for running kaf producer").addArgument(executableCommand).log();
-        ExecResult result = Exec.exec(message, executableCommand, Duration.ofSeconds(30), true, false, null);
-
-        String log = kubeClient().logsInSpecificNamespace(deployNamespace, name);
-        if (result.isSuccess()) {
-            LOGGER.atInfo().setMessage("kaf client produce log: {}").addArgument(log).log();
-        }
-        else {
-            LOGGER.atError().setMessage("error producing messages with kaf: {}").addArgument(log).log();
-            throw new KubeClusterException("error producing messages with kaf: " + log);
-        }
+        KafkaUtils.produceMessagesWithCmd(deployNamespace, executableCommand, message, name, KafkaClientType.KAF.name().toLowerCase());
```

Comment:
Why is it some clients use io.kroxylicious.systemtests.utils.KafkaUtils#produceMessages and some use io.kroxylicious.systemtests.utils.KafkaUtils#produceMessagesWithCmd?  Can we standardise on the former?

Reply:
I couldn't make them work using the same approach than for strimzi test clients. I would need to dedicate more time to go for this. I totally agree that it will be much better to have all the clients in the same way.